src/OpenTelemetry.Instrumentation.StackExchangeRedis/.publicApi/net8.0/PublicAPI.Unshipped.txt

```diff
@@ -0,0 +1,24 @@
+OpenTelemetry.Instrumentation.StackExchangeRedis.StackExchangeRedisInstrumentation
```

Comment:
I think that the contract is common for all targets. This file (and the second one under .ne8 folder) should be removed.

Ref: #1711